frontend/src/components/tab/Tab.styled.ts

```diff
@@ -0,0 +1,64 @@
+import { keyframes, styled } from 'styled-components';
+
+export const TabPageContainer = styled.div`
+  width: 70rem;
+  height: 100%;
+  margin: 0 auto;
+  padding-top: 3rem;
+`;
+
+export const TabListContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const fadeIn = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const TabCurrentContentContainer = styled.div`
+  word-break: break-all;
+  margin-top: 2rem;
+  animation: ${fadeIn} 0.3s ease-out;
+`;
+
+export const TabContainer = styled.div<{ selected: boolean }>`
```

Comment:
```suggestion
export const TabContainer = styled.div<{ isSelected: boolean }>`
```

is 앞에 붙여주면 좋을 것 같습니다 🫡

Reply:
수정하였어요! 짚어주셔서 감사합니다 🙇